promptsource/templates/quac/templates.yaml

```diff
@@ -0,0 +1,90 @@
+dataset: quac
+templates:
+  12c9d007-991c-49ed-82e2-13a7d3147881: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 12c9d007-991c-49ed-82e2-13a7d3147881
+    jinja: "Given the dialogue : \n\nStudent: {{questions[0]}}\n\nTeacher: {{answers.texts[0]\
+      \ | choice}}\n\nThe context : {{context}}\n\nAnswer the question: {{questions[1]\
+      \ }}\n|||\n{{answers.texts[1] | choice }}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Squad
```

Comment:
Just want to check that Squad the metric they use? For some reason I thought this was more complex.

Reply:
According to the leaderboard http://quac.ai/ they use F1, HEQQ and HEQD. Since our answer spans have vaguer boundaries than the shorter ones in SQuAD, exact match is not a useful metric.

> HEQ measures the percentage of examples
> for which system F1 exceeds or matches human
> F1. We compute two variants: (1) the percentage
> of questions for which this is true (HEQ-Q), and
> (2) the percentage of dialogs for which this is true
> for every question in the dialog (HEQ-D).